rules/Renaming/Rector/Name/RenameClassRector.php

```diff
@@ -108,13 +92,7 @@ public function refactor(Node $node): ?Node
         $oldToNewClasses = $this->renamedClassesDataCollector->getOldToNewClasses();
         if ($oldToNewClasses !== []) {
             $scope = $node->getAttribute(AttributeKey::SCOPE);
-            $renameNode = $this->classRenamer->renameNode($node, $oldToNewClasses, $scope);
-
-            if ($renameNode instanceof Namespace_) {
```

Comment:
I am on mobile, but this is needed iirc.

this is needed when we rename ClassLike from no-namespace to namespaced class, then it should be resturctured its stmts, as FileWithoutNamespace_ is returned as [FileWithoutNameapace_], while Namespaced got all stmts from top to down